CMakeLists.txt

```diff
@@ -170,6 +170,8 @@ SET(${PROJECT_NAME}_SOURCES
 ADD_LIBRARY(${PROJECT_NAME} SHARED
   ${${PROJECT_NAME}_SOURCES} ${${PROJECT_NAME}_HEADERS})
 TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC $<INSTALL_INTERFACE:include>)
+TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME}
+  SYSTEM PUBLIC ${Boost_INCLUDE_DIRS})
```

Comment:
In the long term, we should use Boost:: targets, but for now you are right, I think this is useful.